mmf/datasets/builders/glue/builder.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import torch
+from mmf.common.registry import registry
+from mmf.common.sample import Sample
+from mmf.datasets.base_dataset import BaseDataset
+from mmf.datasets.mmf_dataset_builder import MMFDatasetBuilder
+from mmf.utils.general import retry_n
+
+from datasets import load_dataset
```

Comment:
Is `datasets` supported internally in fbcode? Changes in TARGET file are missing if it is supported. Otherwise have to figure out a way to enable this only in OSS.